packages/web3-providers/src/Cloudflare/TokenIconSpecialIconList.json

```diff
@@ -192,6 +192,6 @@
     {
         "name": "AVAX",
         "address": "0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7",
-        "logo_url": "https://raw.githubusercontent.com/sushiswap/icons/master/network/avalanche.jpg'"
+        "logo_url": "https://raw.githubusercontent.com/sushiswap/icons/master/network/avalanche.jpg"
```

Comment:
You can use a CDN link here. Learn more: `https://github.com/DimensionDev/Assets`.